### DISCUSSION TITLE
feature: Homing Projectiles

### DISCUSSION THREAD

**Discussion by Uyarrr** [AscensionGameDev/Intersect-Engine#1548] [Ideas]
**Is your feature request related to a problem? Please describe.**
Right now projectile system lacking homing projectiles that is really important. Developers should be able to make projectiles homing for both spells and basic attacks. This will allow developers to make more fancy combat with them.

**Describe the solution you'd like**
A tickbox for projectile editor to make it homing, which follows selected target. We could also add expiration timer so that, projectile disappears after a while if it does not hit the target.

**Comment by PyroTech03** [comment]
This would be a great option to have.

**Comment by dhmello** [comment]
I don't think they'll ever put this in the engine, unless someone steps up to do it, it's been a request for years.

**Comment by WeylonSantana** [comment]
closing, already added in the engine
